multi/imxrt-multi/pct2075.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Phoenix-RTOS
+ *
+ * i.MX RT integrated PCT2075 driver
+ *
+ * Copyright 2021, 2024 Phoenix Systems
+ * Author: Marek Bialowas, Jacek Maksymowicz
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+
+#include <board_config.h>
```

Comment:
`board_config.h` and `config.h` should not be included directly. Include just `common.h` instead of these two (see: uart.c, spi.c, etc.). This change in headers was introduced due to conflicts between default values and project values not being respected as they should

I missed this in i2c driver

Reply:
OK, fixed this